yarn.lock

```diff
@@ -621,11 +621,6 @@
   dependencies:
     "@sinonjs/commons" "^2.0.0"
 
-"@tsd/typescript@~4.5.2":
```

Comment:
If TypeScript is not installed and `target` version is not specified, TSTyche uses the `latest` TypeScript for testing. Is that right, or do you prefer to test on TS v4.5?

TSTyche can also test on several versions, if that sounds useful.